### DISCUSSION TITLE
[Announcement] HW3 is graded

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs524-program-analysis#263] [Announcements]
Hi all,
Great job on the abstract interpretation.

We have posted your grades in both Gradescope and KLMS.
You can find the detailed criteria for your scores in Gradescope.

**Comment by m-spitfire** [comment]
We are using Gradescope for claim of this homework, right?

**Comment by goodtaeeun** [comment]
You can either use Gradescope's internal system or email the TAs

**Comment by tncks0121** [comment]
저의 재채점 요청이 받아들여져서 Gradescope에서는 점수가 반영되었는데, KLMS에는 반영되지 않았습니다. 문제가 없는지 확인 가능할까요?

My regrading request was accepted and my score was changed accordingly in Gradescope, but the score is not changed in KLMS. Can you check if it's okay?

**Comment by goodtaeeun** [comment]
KLMS 성적을 방금 업데이트 했는데, 확인해보시면 될 것 같습니다.

**Comment by tncks0121** [comment]
반영된 것 확인했습니다. 감사합니다.
